MarsRover.Tests/RoverShould.cs

```diff
@@ -11,29 +11,31 @@ public class RoverShould
         [InlineData(Direction.South, Direction.East)]
         [InlineData(Direction.West, Direction.South)]
         [InlineData(Direction.East, Direction.North)]
-        public void Turn_ReturnsDirection_GivenCommandTurnLeft(Direction direction, Direction expected)
+        public void ExecuteCommand_ReturnsDirection_GivenCommandTurnLeft(Direction direction, Direction expected)
         {
             var command = Command.TurnLeft;
             var rover = new Rover(direction, 2, 2);
+            rover.Direction.Should().Equals(direction);

```

Comment:
this is not required as it's just testing a constructor/property setter & there's no logic involved